### ISSUE TITLE
Rate Us Is Read Incorrectly When Using TalkBack Accessibility Feature

### ISSUE THREAD

**Issue by JB184351** [woocommerce/woocommerce-android#7404]
**Describe the bug**
When Talkback is reading over Rate Us is read as Rate U-S (like United States of America US).

**To Reproduce**
1. Go to Menu
2. Click on Settings
3. Scroll down to `Woo Commerece`
4. Select `Woo Commerece`
5. Activate TalkBack
6. Select the `Rate Us` row
7. Notice how it reads as Rate U-S 

**Screenshots**
<img src="https://user-images.githubusercontent.com/36495451/190716946-da9522ac-256f-4959-847d-ca395099d77f.jpg" width="300"/>

**Expected behavior**
This should be read as Rate Us

**Isolating the problem (mark completed items with an [x]):**
- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Mobile Environment**
Please include:
- Device: Samsung A51
- Android version: 12
- WooCommerce Android version: 10.1
